optimism.tokenlist.json

```diff
@@ -593,6 +593,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x15Ee120fD69BEc86C1d38502299af7366a41D1a6",
+      "name": "BitANT",
+      "symbol": "BitANT",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitANT.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x5029C236320b8f15eF0a657054B84d90bfBEDED3",
```

Comment:
Your token has to implement `IL2StandardERC20` interface otherwise it won't work. See https://community.optimism.io/docs/developers/bridge/standard-bridge.html#adding-an-erc20-token-to-the-standard-bridge